packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -704,13 +704,17 @@ export class RemotedFluidDataStoreContext extends FluidDataStoreContext {
         if (typeof this.initSnapshotValue === "string") {
             const commit = (await this.storage.getVersions(this.initSnapshotValue, 1))[0];
             tree = await this.storage.getSnapshotTree(commit) ?? undefined;
+            this.logger.sendTelemetryEvent({ eventName: "LegacyDataStoreSnapshot", reason: "snapshotAsString" });
         } else {
             tree = this.initSnapshotValue;
         }
 
         const localReadAndParse = async <T>(id: string) => readAndParse<T>(this.storage, id);
         if (tree) {
             const loadedSummary = await this.summarizerNode.loadBaseSummary(tree, localReadAndParse);
+            if (loadedSummary.outstandingOps.length > 0) {
+                this.logger.sendTelemetryEvent({ eventName: "LegacyDataStoreSnapshot", reason: "outstandingOps" });
```

Comment:
@anthony-murphy We disabled writing differential summaries and this code path seems to be for that case right? Any idea if we can add more specific telemetry logs here to ensure we are not still getting older snapshot with differential summary.